hi-ml-histopathology/src/histopathology/utils/naming.py

```diff
@@ -70,3 +70,9 @@ class MetricsKey(str, Enum):
     PRECISION = 'precision'
     RECALL = 'recall'
     F1 = 'f1score'
+
+
+class ModelKey(str, Enum):
```

Comment:
Thanks for introducing the keys. I think `BaseDeepMILModule` still has hardcoded strings for stages. Could you please also change these?

Reply:
done, thanks for the suggestion